### PR TITLE
Bubble-sort algorithm

### PR DESCRIPTION
**Pull request by evabanegacom**
This is an algorithm that sorts arrays in a pair format, comparing the previous element with the next one
we  have
written a bubble sort algorithm for sorting
and using yield as well to compare array characters based on size.

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    len_.times do |x|
+      if yield(array[x], array[x + 1]).positive?
+        array[x], array[x + 1] = array[x + 1], array[x]
+        sorted = true
+      end
+    end
+    len_ -= 1
+    break unless sorted
+  end
+  print array
```
Comment (mosesogwo) [inline bubble_sort.rb]: Your method should return the sorted array and not print it. As it is now, it returns nil.

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+end
+
+x = [4, 5, 3, 7, 6, 8, 96]
+print bubble(x)
+bubble_sort_by(%w[hi hello hey]) do |left, right|
+  left.length - right.length
+end
```
Comment (mosesogwo) [inline bubble_sort.rb]: You can remove these lines of code once you have verified that your methods are working correctly.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @evabanegacom, @karthi07:clap:, Good work on this project. :+1:**

_**Please see my comment on the bubble_sort_by method, make the change, and submit another code review request.**_

**Any question? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by evabanegacom** [comment]
i have fixed the return function issue

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @evabanegacom and @karthi07  :balloon: :lollipop: 

### You've done all the changes and your project works as expected so you may move on Congratulations! :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

PROJECT STATUS: APPROVED :green_apple:

**Pull request merged**